build-parent/pom.xml

```diff
@@ -166,6 +166,29 @@
         </dependencies>
     </dependencyManagement>
 
```

Comment:
I (Informational): context on why this works can be found here: https://codifyiq.substack.com/publish/posts/detail/145767837?referrer=%2Fpublish%2Fposts

We learned this fixing a similar issue on Habushu, so propagating the change here